setup.py

```diff
@@ -52,26 +54,48 @@
 )
 
 
-py_version = sys.version[:3]
+py_version = sys.version_info
+is_pypy = "PyPy" in sys.version
 
 SCRIPT1 = 'nose2'
-SCRIPT2 = 'nose2-%s' % (py_version,)
+SCRIPT2 = 'nose2-%s.%s' % (py_version.major, py_version.minor)
+
 
+def _get_specific_requirements():
+    directory = dirname(__file__)
+    version_specific_requirements_path = 'requirements-{}.{}.txt'.format(py_version.major, py_version.minor)
```

Comment:
Is it possible to make these nested if statements into just one if...elif...elif...else ?
Or maybe something like.
```
if is_pypy:
    pypy_name = "pypy"
else:
   pypy_name = ""
   version_specific_requirements_path = 'requirements-{}{}.{}.txt'.format(pypy_name, py_version.major, py_version.minor)
```
and then rename the requirements files accordingly. 

Overall goal would be to make logic easier to decipher at first glance.

Reply:
because of pypy3 I think you meant : 

```python
pypy_name = ''
if is_pypy:
    pypy_name = "pypy"
version_specific_requirements_path = 'requirements-{}{}.{}.txt'.format(pypy_name, py_version.major, py_version.minor)
```

With that I have to create multiple requirements file for pypy3.3/pypy3.2/pypy3.5, are you ok with that? (pypy doc : http://doc.pypy.org/en/latest/index-of-release-notes.html#cpython-3-5-compatible-versions)

Reply:
The way you ended up doing it looks great and passes travis.